### PR TITLE
Add helper script for rebasing

### PR DESCRIPTION
**Pull request by creydr**
This PR adds a script which automates most of the steps required to rebase up upstream. It has it steps from https://github.com/openshift/kubernetes-nmstate/pull/298.

### CODE REVIEW

File: hack/ocp-rebase.sh [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+# rebase.
+
+set -e
+
+git remote update
+
+upstream_remote=$(git remote -v | grep "https://github.com/nmstate/kubernetes-nmstate (fetch)" | awk '{print $1}')
+downstream_remote=$(git remote -v | grep "https://github.com/openshift/kubernetes-nmstate (fetch)" | awk '{print $1}')
+
+read -p "Soure branch from upstream (usually main): " upstream_source_branch
```
Comment (dougsland) [inline hack/ocp-rebase.sh:16]: :s/Soure/Source/g

File: hack/ocp-rebase.sh [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+echo "Cherry-picked commit since (${last_merge_commit})"
+echo "New merge commit is \"${merge_commit_msg} (${merge_commit})\""
+echo
+echo "Cherry-picked/carried commits:"
+for commit in $(git --no-pager log --oneline --reverse --no-merges ${merge_commit}..${merge_branch} | awk '{print $1}'); do
+    commit_msg=$(git --no-pager log --format=%s -n 1 ${commit})
+    echo "${commit_msg} (${commit})"
+done
+
+echo "Please make sure the commits have the \"USTREAM: <carry>:\" prefix and check if you can squash commits"
```
Comment (dougsland) [inline hack/ocp-rebase.sh:70]: :s/USTREAM/UPSTREAM/g

**Review by dougsland — COMMENTED** [review]
nit comment, lgtm

File: hack/ocp-rebase.sh (hunk 1 above)
Comment (creydr) [reply hack/ocp-rebase.sh]: good catch :+1: 
updated it

File: hack/ocp-rebase.sh (hunk 2 above)
Comment (creydr) [reply hack/ocp-rebase.sh]: good catch :+1: 
updated it

**Comment by creydr** [comment]
The ovn e2e handler test failures should not be related to this PR as this adds only a new helper script which is not used in CI.

**Comment by creydr** [comment]
@cybertron @dougsland could you PTAL?

**Comment by creydr** [comment]
@cybertron @dougsland could you PTAL?

**Comment by cybertron** [comment]
/lgtm

I was hoping to try this out myself, but since I keep failing to find time for that let's just get it in. Thanks!

**Pull request merged**